spec/models/application_record_spec.rb

```diff
@@ -104,6 +104,21 @@
     end
   end
 
+  describe 'interview_count' do
+    before :each do
+      record_1 = create :application_record
+      record_2 = create :application_record
+      create :interview, application_record: record_1
+      @collection = ApplicationRecord.where(id: record_1.id)
```

Comment:
But then wouldn't this code break? I thought calling a model method on an ActiveRecord relation that contains more than one object will throw an exception because it's like trying to call `["1", "2"].to_i`.